apps/moose-console/src/lib/snippets.ts

```diff
@@ -0,0 +1,109 @@
+import { CliData, DataModel, Route, Table, column_type_mapper } from "app/db";
+import { getIngestionPointFromModel } from "./utils";
+
+export const jsSnippet = (data: CliData, model: DataModel) => {
+    const ingestionPoint = getIngestionPointFromModel(model, data);
+
+
+    const columns = model.columns.map((field, index) => {
+        const data_type = column_type_mapper(field.data_type);
+
+        if (data_type === "string") {
+            return `{${field.name}: "test-value${index}"}`
+            } else if (data_type === "number") {
+            return `{${field.name}: ${index}}`
+            } else if (data_type === "boolean") {
+            return `{${field.name}: ${index % 2 === 0}}`
+            } else if (data_type === "date") {
+            return `{${field.name}: "2022-01-01"}`
+            } else if (data_type === "array") {
+            return `{${field.name}: ["test-value${index}"]}`
+            } else if (data_type === "object") {
+            return `{${field.name}: { key: "test-value${index}" }}`
+            }
```

Comment:
The above lines look copy pasted between the JS and Python snippets - could we have it as a function?